src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/version-switcher.html

```diff
@@ -3,11 +3,11 @@
 <script>
 document.write(`
   <div class="version-switcher__container dropdown">
-    <button type="button" class="version-switcher__button btn btn-sm navbar-btn dropdown-toggle" data-bs-toggle="dropdown">
+    <button id="versionswitcherbutton" type="button" class="version-switcher__button btn btn-sm navbar-btn dropdown-toggle" data-bs-toggle="dropdown" aria-haspopup="true" aria-controls="versionswitchermenu">
       {{ theme_switcher.get('version_match') }}  <!-- this text may get changed later by javascript -->
       <span class="caret"></span>
     </button>
-    <div class="version-switcher__menu dropdown-menu list-group-flush py-0">
+    <div id="versionswitchermenu" class="version-switcher__menu dropdown-menu list-group-flush py-0" role="menu" aria-labelledby="versionswitcherbutton">
```

Comment:
@trallard here the inner div (the thing that pops open when the button is clicked) gets an ID, `role='menu'` and an `aria-labelledby` (again I'm copying from MDN example, not certain I've done this optimally)

Reply:
There is another `aria` property needed here [`aria-activedescendant`](https://www.w3.org/TR/wai-aria-1.2/#aria-activedescendant) which I assume will be related to `activeVersionName`, but I am unsure how to add this (seems to be dynamically set in the JS file)

Reply:
> There is another `aria` property needed here [`aria-activedescendant`](https://www.w3.org/TR/wai-aria-1.2/#aria-activedescendant) which I assume will be related to `activeVersionName`, but I am unsure how to add this (seems to be dynamically set in the JS file)

I'm actually unsure about this.  IIUC, `aria-activedescendant` is about the element *that has DOM focus* whereas `activeVersionName` is about the version of the docs that is currently loaded in the browser (regardless of which version *choice* the user happens to have tabbed to when preparing to switch versions).  Is that right?  If so / if I'm reading [the w3c page](https://www.w3.org/TR/wai-aria-1.2/#aria-activedescendant) correctly, we don't want to set `aria-activedescendant` equal to `activeVersionName`, and we don't actually *need* to set `aria-activedescendant` at all since we set `tabindex=-1` (therefore the container is not focusable).  Does that sound right to you @trallard or am I misunderstanding?